Package.swift

```diff
@@ -22,7 +22,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", .exact("2.11.0-rc.1")),
+        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", from: "2.10.0"),
```

Comment:
I guess we should also update versions in [CHANGELOG](https://github.com/mapbox/mapbox-navigation-ios/blob/29cda19265b233b3a7c5d89716403ecb3741cad4/CHANGELOG.md#packaging)